webapp/portlet/src/main/webapp/activity-composer-app/components/ExoActivityComposer.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="activityComposer">
+    <v-app id="activityComposerApp" class="VuetifyApp">
+      <div class="openLink">
+        <a @click="showMessageComposer = true">
+          <i class="uiIconPlus"></i>{{ $t('activity.composer.link').replace('{0}', postTarget) }}
+        </a>
+      </div>
+
+      <v-navigation-drawer
+        v-model="showMessageComposer"
+        right
+        fixed
+        temporary
+        width="420"
```

Comment:
The `width` should be in percentage
```suggestion
        width="33%"
```